src/main/java/org/spongepowered/api/Server.java

```diff
@@ -298,12 +298,12 @@
     int getRunningTimeTicks();
 
     /**
-     * Get the sink that messages to be broadcast across the whole server
+     * Get the channel that messages to be broadcast across the whole server
      * should be sent to.
      *
-     * @return The server-wide broadcast sink
+     * @return The server-wide broadcast channel
      */
-    MessageSink getBroadcastSink();
+    MessageChannel getBroadcastChannel();
```

Comment:
I'm not sure about that, but this could probably be defaulted to `MessageChannels.TO_ALL`.